packages/lesswrong/components/common/useCheckMeritsCollapse.ts

```diff
@@ -0,0 +1,22 @@
+import { RefObject, useEffect, useRef, useState } from 'react';
+
+export function useCheckMeritsCollapse<T extends HTMLElement>(
```

Comment:
This was pulled out of `EAUsersProfileTabbedSection` so we could use it elsewhere